recipes-kernel/linux/linux-base_git.bbappend

```diff
@@ -11,3 +11,5 @@ LINUX_DEFCONFIG_raspberrypi3-64 = "defconfig"
 KERNEL_IMAGETYPE_raspberrypi3-64 = "Image"
 KERNEL_DEVICETREE_raspberrypi3-64 = "broadcom/bcm2837-rpi-3-b-plus.dtb \
                                      broadcom/bcm2837-rpi-3-b.dtb"
+
+CVE_VERSION = "4.19.132"
```

Comment:
Umm, I don't want this recipe to have the specific version(x.y.z). I need to come up with some idea...

Reply:
Could you try if the following statement works well?

```
[conf/distro/emlinux.conf]
 LINUX_GIT_SRCREV ?= "4da95b68eb2b04de4ec212c17db4863d0301e7f9"
+CVE_VERSION ?= "${@bb.utils.contains('LINUX_GIT_SRCREV', '4da95b68eb2b04de4ec212c17db4863d0301e7f9', '4.19.132', '${PV}', d)}"
```

Reply:
Could you try if the following statement works well?

```
[conf/distro/emlinux.conf]
 LINUX_GIT_SRCREV ?= "4da95b68eb2b04de4ec212c17db4863d0301e7f9"
+LINUX_CVE_VERSION ?= "${@bb.utils.contains('LINUX_GIT_SRCREV', '4da95b68eb2b04de4ec212c17db4863d0301e7f9', '4.19.132', '${PV}', d)}"

[recipes-kernel/linux/linux-base_git.bbappend]
+CVE_VERSION = "${LINUX_CVE_VERSION}"
```

Reply:
I confirmed your suggestion works well and re-pushed fixed commits.